src/content/subgames.cpp

```diff
@@ -37,7 +37,22 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 bool getGameMinetestConfig(const std::string &game_path, Settings &conf)
```

Comment:
also if you're touching this anyway can you make the function static?
it's only used inside subgames.cpp yet global and also defined in the `.h`

Reply:
I'd also prefer to return `Settings` here directly, because `readConfigFile` is cumulative and wouldn't work as intended if used on `g_settings`.

Reply:
It's only used in subgames.cpp, so I could make it private instead - ie: in an anonymous namespace